assets/js/src/common/form/checkbox/_stories/checkbox.tsx

```diff
@@ -19,14 +19,14 @@ export const Checkboxes = () => (
     <Heading level={3}>Inline individual checkboxes</Heading>
     <div>
       <Checkbox
-        onChange={action('checkbox-individual-1')}
+        onCheck={action('checkbox-individual-1')}
```

Comment:
`onChange` is a native HTML attribute and its type requires the callback parameter to be an event (https://github.com/DefinitelyTyped/DefinitelyTyped/blob/9a6908d05cccf32a073e93e3c39f1df3858b435f/types/react/index.d.ts#L2097). But it's harder to work with that because then you need to get the value from an event. So I used `onCheck` (which we already use in https://github.com/mailpoet/mailpoet/blob/master/assets/js/src/common/toggle.tsx#L5 - this component will be deleted soon) so we can return meaningful value/values.